tests/test_command.py

```diff
@@ -428,10 +428,27 @@ def test_submit_notebook_with_program(self):
             qds.main()
 
     def test_submit_script_location_aws(self):
```

Comment:
@tarrygo : Also please add tests for aws scala, R and sql just like we have tests for local script location.